content/manager/bootstrapping.md

```diff
@@ -165,6 +165,153 @@ Services:
+    {{% /gsNote %}}
+
+ 1. Make a note of the IP/hostname
+
+{{< gsHighlight  sh  >}}
+    $ CLOUDIFY_HOST={your-manager-public-ip}
+{{< /gsHighlight >}}
+
+
+## Web UI
+
+ 1. In your browser, navigate to http://{your-manager-public-ip}
+
+    {{% gsNote %}}
+    The manager VM may take some time to start up, so if it doesn't load immediately don't worry.
+    {{% /gsNote %}}
+
+    {{% gsNote %}}
+    If security is enabled, your browser will complain that the TLS/SSL certificate is not valid. Temporarily allow the connection: A new certificate will be generated as part of this setup process.
+    {{% /gsNote %}}
+
+ 2. If asked, authenticate using `cloudify` as both the username and password.
+
+ 3. Create a new deployment of the `CloudifySettings` blueprint:
+    ![CloudifySettings blueprint deploy]({{< img "manager/image-deploy-new-deployment.png" >}})
+
+ 5. Fill in the required fields:
+
+    | name                            | platform  | secure only | notes                                                                                                                                                                                                                                                                    |
+    | -----                           | -----     | -----       | -----                                                                                                                                                                                                                                                                    |
+    | user_ssh_key                    |           |             | This will be added to ~/.ssh/authorized_keys in addition to any key you specified when initializing the instance.                                                                                                                                                        |
+    | agents_security_group_name      |           |             | Security group to be generated and used for accessing agents on VMs. This must not alerady exist as the `install` workflow will create a new one                                                                                                                         |
+    | agents_keypair_name             |           |             | Keypair to be generated and used for accessing agents on VMs. This must not alerady exist as the `install` workflow will create a new one.                                                                                                                               |
+    | agents_user                     |           |             | User to be used for accessing agents on VMs. This should be `centos`.                                                                                                                                                                                                    |
```

Comment:
It *is* centos unless you've built your own manager image, but yeah I'll make it a little less definite.